content/en/docs/setup/install/multicluster/single-network/remote/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Add a Remote Cluster
+description: Describes how to add a remote cluster to your mesh on a flat network.
+weight: 30
+keywords: [multicluster, remote-cluster, deployment]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, you can add a
+{{< gloss >}}remote cluster{{< /gloss >}} to your
+{{< gloss >}}service mesh{{< /gloss >}}.
+
+To give you clean instructions, this section starts right after you complete the
+initial configuration, and assumes that `Cluster_1` and `Cluster_2` reside on
+the same network. The new remote cluster, `Cluster_2` in this section, shares
+the {{< gloss >}}control plane{{< /gloss >}} of `CLUSTER_1`, and this guide
+assumes that `Cluster_1` and `Cluster_2` reside on the same network.
+
+Production systems use this configuration when all clusters within a region
+share a common control plane. Any configuration with one control plane per zone
+
+The following diagram shows a multicluster deployment with a
+{{< gloss >}}primary cluster{{< /gloss >}} and a remote cluster:
+
+{{< image width="75%"
+    link="remote.svg"
+    caption="A multicluster deployment with a primary and a remote cluster"
+    >}}
+
+Complete the [initial configuration instructions](/docs/setup/install/multicluster/single-network/initial-configuration)
+before you continue.
+
+## Configure Trust
+
+{{< boilerplate trust-config >}}
+
+## Deploy Istio
+
+Next, deploy Istio in `Cluster_2` with the discovery address pointing at the
+ingress gateway of `Cluster_1`. Your [initial configuration](/docs/setup/install/multicluster/single-network/initial-configuration)
+enabled mesh expansion on `Cluster_1`, and now `Cluster_2` can access the
+discovery server in `Cluster_1` via the ingress gateway.
```

Comment:
This is all on the same network, so no ingress gateway needed, no?  It's not on your diagram

Reply:
For remote, we're using ingress for LB. Otherwise, we have to use pod IP which is volatile. We should make that more clear.

Reply:
I could add the gateway to the diagram and further explanation to the paragraph but doing so adds a lot of technical nuance that readers don't need to know to complete the instructions. I can add a link to the gateway concept page for readers that need more information.